teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
Same here. Please pass the `age, `name` and `parental_permission` into the `initialize` constructor and then use these parameters in the `super`.

Reply:
- [ ] Well resolved 👍🏽